cheat_sheets/pyfluent_cheat_sheet/pyfluent_cheat_sheet.tex

```diff
@@ -0,0 +1,197 @@
+\documentclass[9pt,landscape]{article}
+\usepackage{{./../static/style}}
+\pdfinfo{
+  /Title (PyAnsys Cheat Sheet)
+  /Creator (TeX)
+  /Producer (pdfTeX 1.40.0)
+  /Author (Ansys)
+  /Subject (PyAnsys)
+  /Keywords (PyAnsys, Cheat sheet, template)}
+
+\begin{document}
+\raggedright
+\footnotesize
+
+% Add the title of cheat sheet here
+% ----------------------------------------
+
+\begin{center}
+     \Huge{\textbf{Cheat sheet for PyFluent}} \\
+\end{center}
+\begin{center}
+     \Large{\textbf{Solver Settings Object Interface}} \\
+\end{center}
+
+\AddToShipoutPicture*
+  {\put(670,577.5){\includegraphics[height = 1.2cm]{ansys.png}}}
+\AddToShipoutPictureBG*{\includegraphics[width=\paperwidth]{bground.png}}
+\vspace{-0.15cm}
+\noindent\makebox[\linewidth]{\rule{\paperwidth}{2pt}}
+
+\begin{multicols}{3}
+\setlength{\premulticols}{1pt}
+\setlength{\postmulticols}{1pt}
+\setlength{\multicolsep}{1pt}
+\setlength{\columnsep}{2pt}
+
+% session starts here. 
+% First colomn
+% --------------------------------------------------------------------------------
+\vfill
+\section{\includegraphics[height=\fontcharht\font`\S]{slash.png} Launch Fluent locally}
+Following method to start Fluent from Python in gRPC mode.\\
+This code starts Fluent in the background so that commands can be sent to Fluent from the Python interpreter.
```

Comment:
```suggestion
The following method is used to start Fluent from Python in gRPC mode.\\
This code starts Fluent in the background so that commands can be sent to Fluent from the Python interpreter.
```